### ISSUE TITLE
[PRE REVIEW]: Genotify: Fast, lightweight gene lookup and summarization

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#880]
**Submitting author:** @j-andrews7 (<a href="http://orcid.org/0000-0002-0780-6248">Jared Andrews</a>)
**Repository:** <a href="https://github.com/j-andrews7/Genotify" target ="_blank">https://github.com/j-andrews7/Genotify</a>
**Version:** v1.2.0
**Editor:** @pjotrp
**Reviewer:** Pending

**Author instructions**

Thanks for submitting your paper to JOSS @j-andrews7. The JOSS editor (shown at the top of this issue) will work with you on this issue to find a reviewer for your submission before creating the main review issue.

@j-andrews7 if you have any suggestions for potential reviewers then please mention them here in this thread. In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission.

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon. I'm here to help you with some common editorial tasks. @pjotrp it looks like you're currently assigned as the editor for this paper :tada:

For a list of things I can do to help you, just type:

```
@whedon commands
```

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by pjotrp** [comment]
This is potentially an interesting contribution - especially since it uses Uniprot. The git repo is pretty large, it would be worth throwing out some large files from history (not so light-weight as it needs to be I mean). The species.json file is massive - would it not be better to generate that too? 

Question: do you have the website running somewhere so I can try? It does not say in the README.

**Comment by j-andrews7** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[--> Check article proof :page_facing_up: <--](https://github.com/openjournals/joss-papers/blob/joss.00880/joss.00880/10.21105.joss.00880.pdf)

**Comment by j-andrews7** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[--> Check article proof :page_facing_up: <--](https://github.com/openjournals/joss-papers/blob/joss.00880/joss.00880/10.21105.joss.00880.pdf)

**Comment by serine** [comment]
@j-andrews7 @pjotrp This looks pretty neat ! worked out of the box in first attempt. not sure of the usage/target audience yet, but certainly great aggregate of things and as an exploratory tools will be great resource. 

Having this as a stand along electron app is initial wow factor (at least for me), but not sure sure if this is much of advantage over website/webapp, seems like webapp would have been even more accessible. 

Just a comment; paper pdf feels long, while all information is very relevant and useful, perhaps fair chunk can be moved into general docs. Like I said I don't actually know what JOSS policies are. I thought paper.md was meant to abstract like. @arfon is this so?

In summary I'm rather interested in reviewing this paper further. 

Cheers,

**Comment by serine** [comment]
I should probably add that those comments were rather general and don't directly apply to the review process. From what I understand the review process more or less whether the tools installs and works as described. And it did install and worked. I just need to play with it to appreciate the usability and applicability of the tool. 

One thing that stood out from reading the paper and the github; "gene annotation" seems to be used a few times. To me gene annotation is annotation of a new - unknown region of the genome with a specific name/feature e.g protein_coding gene. In my head gene annotation usually requires two things 1. the data e.g RNAseq and 2. the genome browser of some sort to browser through the genome and looking for new pile ups of reads where there is no known feature, thereby annotating that region as new_feature_A. 

Cheers

**Comment by arfon** [comment]
> Just a comment; paper pdf feels long, while all information is very relevant and useful, perhaps fair chunk can be moved into general docs. Like I said I don't actually know what JOSS policies are. I thought paper.md was meant to abstract like. @arfon is this so?

This paper is certainly longer than most but I think it's OK. The 1-page figure makes it seem quite long (and might be more appropriate in the docs).

**Comment by j-andrews7** [comment]
@serine Thanks for the comments. I'm glad it's working well for you. As for length, the guidelines said 250-1000, I think the paper as it currently stands is ~700. I could probably cut it down a bit, I just wanted to establish the need for it well. I'm not sure if I can really resize the figure in Rmarkdown given that I want to keep the high resolution, but I can certainly move it to the docs. 

The reason I didn't build it as a web app was so I could hotkey it so that it'd be quicker to run through a column of gene names in excel. I was also trying to design it so it could be used on a quarter/half-screen side by side a table or another browser for those of us stuck with single monitor setups. Not having to host the website is also a plus, and I wanted to take a crack at what Electron could do.

As for the "gene annotation" phrase, I kind of agree, but I saw it used in a similar context several other places (even from places like NCBI and Uniprot).
